MoreLinq/SkipUntil.cs

```diff
@@ -61,14 +61,19 @@ public static IEnumerable<TSource> SkipUntil<TSource>(this IEnumerable<TSource>
             {
                 using var iterator = source.GetEnumerator();
 
-                while (iterator.MoveNext())
+                for(;;)
```

Comment:
```suggestion
                for (;;)
```

Reply:
It also can be a. do/while loop.
Anyway, never to late to ask, is there any formating rules description somewhere ? I use resharper for auto-formating and I didn't customize the settings...
In the same fashion of questions : `for(;;)` or `while(true)`

Reply:
Nothing document but hope fd32a8ad5754aea1288c277dedb927053e1c8368 helps as a starting point. Other than that, readability comes first (even if it means breaking rules), then consistency and finally general C# guidelines. Unfortunately, this means I have to switch off all auto-formatting in the IDE; prefer to write for humans, not machines.

> In the same fashion of questions : `for(;;)` or `while(true)` ?

Either is fine as long as there is as space after `for` or `while`:

https://github.com/morelinq/MoreLINQ/blob/fd32a8ad5754aea1288c277dedb927053e1c8368/.editorconfig#L51